pkg/restore/restore.go

```diff
@@ -742,8 +742,90 @@ func (ctx *restoreContext) processSelectedResource(
 				// have to try to create them multiple times.
 				existingNamespaces.Insert(targetNS)
 			}
-			// For namespaces resources we don't need to following steps
+			// For namespaces resources we restore metadata when existing resource policy is update, else continue
 			if groupResource == kuberesource.Namespaces {
+				if existingNamespaces.Has(targetNS) {
```

Comment:
I think we can make condition like 
```
if existingNamespaces.Has(targetNS) && velerov1api.PolicyTypeUpdate == ctx.restore.Spec.ExistingResourcePolicy {...}
```
and ignore printing the "Skipping ...." log message.

Reply:
Accepted. Will make the change.